crime-applications-adaptor/api-tests/src/test/resources/features/GetApplicationData.feature

```diff
@@ -2,12 +2,18 @@ Feature: Retrieve application data from the Crime Apply Datastore.
 
   Scenario Outline: Retrieve valid application data from Crime Apply Data Store by USN.
     Given an application with usn <usn> exists in the datastore
+    And no entry for usn <usn> should be present in the EFORMS_STAGING table
+    And no entry for usn <usn> should be present in the EFORMS_HISTORY table
     When the GET internal V1 crimeapply endpoint is called with usn <usn> and user "<user>"
     Then the returned response should match the contents of "<expectedResponseFile>"
+    And an entry for usn <usn> should have been created in the EFORMS_STAGING table
+    And the entry in the EFORMS_STAGING table for usn <usn> should have no maatRef
+    And an entry for usn <usn> should have been created in the EFORMS_HISTORY table
 
     Examples:
       | scnNo | usn      | user    | expectedResponseFile |
       | 00    | 10000987 | test-a1 | 10000987.json        |
```

Comment:
What is the difference between these 2 scenarios? I would rename the expected Json files or otherwise to make it clear.
It looks like 10000987 is `"meansPassport": "on_benefit_check" `while 10000988 is not?

Reply:
Makes sense - I've updated these file names.